data/json/items/comestibles/veggy_dishes.json

```diff
@@ -300,7 +302,8 @@
     "price": 50,
     "material": "veggy",
     "volume": 1,
-    "charges": 1
+    "charges": 1,
+    "vitamins": [ ["vitA", 83 ], ["vitC", 14 ], ["calcium", 4 ], ["iron", 8 ] ]
```

Comment:
this shouldn't be necessary because of copy-from

Reply:
Removed, fixed.